examples/model_selection/plot_permutation_tests_for_classification.py

```diff
@@ -101,10 +102,10 @@
 #
 # Below we plot the null distribution for the randomized data. The permutation
 # scores are similar to those obtained using the original iris dataset
-# because the permutation always destroys any feature label dependency present.
-# The score obtained on the original randomized data in this case though, is
-# very poor. This results in a large p-value, confirming that there was no
-# feature label dependency in the original data.
+# because the permutation always destroys any feature-label dependency present.
+# The score obtained on the randomized data without label permutation in this case
+# though, is very poor. This results in a large p-value, confirming that there was no
+# feature-label dependency in the randomized data before labels where permuted.
```

Comment:
Here I'm trying to not use the word "original", because it can be confused with the iris data that is called "original data" in the other parts of the example.

Reply:
Yeah good point, maybe we could also amend the legend in the plots, which both say 'original', we could change to 'original iris data' and 'original random data' ?